string message
bool repeat_previous 
---
bool uttered